prime-router/build.gradle.kts

```diff
@@ -53,7 +53,11 @@ val azureAppName = "prime-data-hub-router"
 val azureFunctionsDir = "azure-functions"
 val primeMainClass = "gov.cdc.prime.router.cli.MainKt"
 val defaultDuplicateStrategy = DuplicatesStrategy.WARN
-azurefunctions.appName = azureAppName
+azurefunctions {
+    appName = azureAppName
```

Comment:
`pricingTier` and `region` are either newly required or poorly handled if they don't exist.